jackal_control/launch/control.launch

```diff
@@ -1,10 +1,10 @@
 <launch>
-  <rosparam command="load" file="$(find jackal_control)/config/control.yaml" />
+  <rosparam file="$(find jackal_control)/config/control.yaml" command="load"/>
 
   <node name="controller_spawner" pkg="controller_manager" type="spawner"
         args="jackal_joint_publisher jackal_velocity_controller --shutdown-timeout 1" />
-  
-  <node pkg="robot_localization" type="ekf_localization_node" name="ekf_localization" /> 
+
+  <include file="$(find jackal_control)/launch/robot_localization.launch"/>
```

Comment:
Yeah, let's pull it out. Unless you feel strongly, I'd prefer to just call it `localization.launch`. It's the only one running persistently— anything providing `map -> odom` or `world -> odom` will be either managed by capabilities, or manually launched by the user. And in any case, if those did end up in this package in the future, they could be called `world_localization.launch` for GPS or `amcl_localization.launch` for amcl + ekf.